src/os/bluestore/BlueStore.cc

```diff
@@ -1045,7 +1045,15 @@ struct LruOnodeCacheShard : public BlueStore::OnodeCacheShard {
     --num_pinned;
     dout(20) << __func__ << this << " " << " " << " " << o->oid << " unpinned" << dendl;
   }
-
+  void _rm_pinned(BlueStore::Onode* o) override
+  {
+    ceph_assert(!o->pinned);
```

Comment:
Given the function name this assertion looks a bit confusing. May be remove it totally similar to _unpin()? 
And rename to _unpin_and_rm()?
Not a big deal though...

Reply:
@ifed01 Confusion is bad. I renamed it and deleted assert.